src/bin/cargo-ziggy/main.rs

```diff
@@ -217,6 +213,10 @@ pub struct Run {
     /// Build with ASAN (nightly only)
     #[clap(long = "asan", action)]
     asan: bool,
+
+    /// Stop the run after the first crash is encountered
+    #[clap(short = 'x', long)]
```

Comment:
why -x? 'S' or 'C' might be more intuitive maybe? also what about timeouts?

Reply:
-x kind of looks like a line crashing :smile: I'm open to changing it to `-C`.

`ziggy run` reacting to timeouts is a good idea, I'll create an issue :+1: